components/style/servo/selector_parser.rs

```diff
@@ -100,44 +100,55 @@ impl PseudoElement {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[allow(missing_docs)]
 pub enum NonTSPseudoClass {
-    AnyLink,
-    Link,
-    Visited,
     Active,
+    AnyLink,
+    Checked,
+    Disabled,
+    Enabled,
     Focus,
     Fullscreen,
     Hover,
-    Enabled,
-    Disabled,
-    Checked,
     Indeterminate,
-    ServoNonZeroBorder,
+    Lang(String),
```

Comment:
I'm pretty sure you can if you do: `parser.expect_ident_or_string().into_owned().into_boxed_str()`: https://play.rust-lang.org/?gist=184a2e91d53c9bebfd852ec73ccc34a6&version=stable&backtrace=0